File: frameworks/Java/fair/java-kora-jdbc/src/main/java/io/techempower/benchmark/kora/model/World.java
package io.techempower.benchmark.kora.model;

import io.koraframework.database.common.annotation.Column;
import io.koraframework.database.common.annotation.Id;
import io.koraframework.database.common.annotation.Table;
import io.koraframework.database.jdbc.EntityJdbc;
import io.koraframework.json.common.annotation.Json;

@Json
@EntityJdbc
public record World(@Id @Column("id") int id,
                    @Column("randomnumber") int randomNumber) {
}

File: frameworks/Kotlin/kora/kora-jdbc-coroutines/src/main/kotlin/io/koraframework/benchmark/model/World.kt
package io.koraframework.benchmark.model

import ru.tinkoff.kora.database.common.annotation.Column
import ru.tinkoff.kora.database.common.annotation.Id
import ru.tinkoff.kora.database.jdbc.EntityJdbc
import ru.tinkoff.kora.json.common.annotation.Json

@Json
@EntityJdbc
data class World(
    @field:Id @field:Column("id") val id: Int,
    @field:Column("randomnumber") val randomNumber: Int,
)
